tests/training/test_model_trainer.py

```
@@ -323,43 +327,40 @@ def test_trainer_litdata(config, tmp_path: str):
 def test_trainer_torch_dataset(config, tmp_path: str):
+    OmegaConf.update(config, "data_config.data_pipeline_fw", "torch_dataset")
     OmegaConf.update(config, "trainer_config.save_ckpt_path", None)
-    model_trainer = ModelTrainer(config, data_pipeline_fw="torch_dataset")
+    model_trainer = ModelTrainer(config)
     assert model_trainer.dir_path == "."
 
     ##### test for reusing np chunks path
+    OmegaConf.update(config, "data_config.data_pipeline_fw", "torch_dataset_np_chunks")
+    OmegaConf.update(config, "data_config.np_chunks_path", tmp_path)
+    OmegaConf.update(config, "data_config.use_existing_np_chunks", True)
     with pytest.raises(Exception):
```

Ruff (0.8.2)

`pytest.raises(Exception)` should be considered evil

(B017)